### PR TITLE
Fix:Travis Build Error

### PR DESCRIPTION
**Pull request by aayushsingla**
Fixes: Travis Build Failing

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by aayushsingla** [comment]
@therajanmaurya @miPlodder please review this PR as this is a basic issue and other pull requests must rebase on this to get merged.

File: app/src/androidTest/java/org/mifos/mobile/ApplicationTest.java [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
-import static org.junit.Assert.assertEquals;
-
 import android.content.Context;
-import androidx.test.InstrumentationRegistry;
-import androidx.test.runner.AndroidJUnit4;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import androidx.test.InstrumentationRegistry;
```
Comment (prashantkh19) [inline app/src/androidTest/java/org/mifos/mobile/ApplicationTest.java:8]: Why did you delete and re-write the same code? Make sure your PR contains the changes only.

**Comment by prashantkh19** [comment]
@therajanmaurya @geekanamika I think we should merge this first and rebase all the PRs on it.

**Review by prashantkh19 — APPROVED** [review]
